src/tests/ftest/security/pool_security_acl.py

```diff
@@ -0,0 +1,421 @@
+                    username = found_user.group(1)
+                    userpermission = found_user.group(2)
+                    pool_permission_list.append(line)
+            elif line.startswith("A:G:"):
+                found_group = re.search(r"A:G:(.+)@:(.*)", line)
+                if found_group:
+                    groupname = found_group.group(1)
+                    grouppermission = found_group.group(2)
+                    pool_permission_list.append(line)
+        return pool_permission_list
+
+    def verify_daos_pool_result(self, result, action, expect, err_code):
+        '''
+        Deascription:
+            To verify the daos pool read or write action result.
+        Args:
+            result: handle for daos pool action.
+            action: daos pool read or write.
+            expect: expecting pass or deny.
+            err_code: expecting deny of RC code.
+        Return:
+            none.
+        '''
+        if expect.lower() == 'pass':
+            if result.exit_status != 0 or result.stderr != "":
+                self.fail("##Test Fail on verify_daos_pool {}, "
+                          "expected Pass, but Failed.".format(action))
+            else:
+                self.log.info(" =Test Passed on verify_daos_pool {}, "
+                              "Succeed.\n".format(action))
+        elif err_code not in result.stderr:
+            self.fail("##Test Fail on verify_daos_pool {},"
+                      " expected Failure of {}, but Passed."
+                      .format(action, expect))
+        else:
+            self.log.info(" =Test Passed on verify_daos_pool {},"
+                          " expected error of {}.\n"
+                          .format(action, expect))
+
+    def verify_pool_readwrite(self, svc, uuid, action, expect='Pass'):
+        '''
+        Deascription:
+            To verify client is able to perform read or write on a pool.
+        Args:
+            svc:  pool svc number.
+            uuid: pool uuid number.
+            action: read or write on pool.
+            expect: expecting behavior pass or deny with RC -1001.
+        Return:
+            pass or fail.
+        '''
+        deny_access = '-1001'
+        daos_cmd = DaosCommand(os.path.join(self.prefix, "bin"))
+        if action.lower() == "write":
+            daos_cmd.request.value = "container"
+            daos_cmd.action.value = "create --svc={} --pool={}"\
+                                     .format(svc, uuid)
+        elif action.lower() == "read":
+            daos_cmd.request.value = "pool"
+            daos_cmd.action.value = "query --svc={} --pool={}"\
+                                     .format(svc, uuid)
+        else:
+            self.fail("##In verify_pool_readwrite, invalid action: {}"
+                      .format(action))
+        daos_cmd.exit_status_exception = False
+        result = daos_cmd.run()
+        self.log.info("  In verify_pool_readwrite {}..".format(action))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments